src/chainparams.cpp

```diff
@@ -298,3 +320,24 @@ bool SelectParamsFromCommandLine()
     SelectParams(network);
     return true;
 }
+
+
+// Block height must be >0 and less than the halving interval block height
+// Index variable i ranges from 0 - (vFoundersRewardAddress.size()-1)
+std::string CChainParams::GetFoundersRewardAddress(size_t nHeight) const
+{
+    assert(nHeight>0 && nHeight<consensus.nSubsidyHalvingInterval);
+    size_t i = (size_t)floor((double(nHeight)/consensus.nSubsidyHalvingInterval)*vFoundersRewardAddress.size());
```

Comment:
I liked @Daira's formula of `nHeight / ceiling(consensus.nSubsidyHalvingInterval / vFoundersRewardAddress.size())`.One nice quality of it is that `ceiling(consensus.nSubsidyHalvingInterval / vFoundersRewardAddress.size())` does not vary over runtimes, and we might be able to define it as a const.